### DISCUSSION TITLE
Deploy Imagemagick Container in Kubernetes

### DISCUSSION THREAD

**Discussion by lalitap25** [ImageMagick/ImageMagick#6806] [Development]
### ImageMagick version

7.1.1.1

### Operating system, version and so on

Linux

### Description

Hi There, 

I wan tto use docker imgae of Imagemagick to deploy it as a container in kubernetes. But currently usign deployment.yaml file, I get error crashloopbackoff error. My node has all the resource. I checked the logs also
kubectl logs pod_name 
I got Imagemagick version. And no error message.
Version: ImageMagick 7.1.1-13 Q16-HDRI x86_64 21276 https://imagemagick.org/
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules 
Delegates (built-in): bzlib cairo fontconfig freetype gslib heic jng jpeg jxl lcms ltdl lzma png ps raw rsvg tiff webp x xml zlib
Compiler: gcc (12.2)

Can someone tell me why I ma not able to deploy the container.

**Comment by dlemstra** [comment]
Can you run the container image locally?
